### PR TITLE
py-me-al: Submission

### PR DESCRIPTION
**Pull request by essandess**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS x.y
Xcode x.y / Command Line Tools x.y.z

###### Verification 
Have you

- [ ] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [ ] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [ ] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [ ] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by reneeotten** [comment]
@essandess this seems a very inactive project that didn't have any releases and is missing metadata like `license`. Do we really need this for anything ;) ?

**Comment by essandess** [comment]
This PR will give MacPorts that little extra püsh. https://www.youtube.com/watch?v=uMSV4OteqBE

File: python/py-me-al/Portfile [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        dabeaz me-al 0f2e04aecff88b73a25fe69e789f3ade26683866
+name                py-${github.project}
+version             2018.11.24
+revision            0
+categories-append   lang
+license             none
```
Comment (reneeotten) [inline python/py-me-al/Portfile:12]: I don't think this is a valid value... one could/should ask upstream what their intended license is

Comment (essandess) [reply python/py-me-al/Portfile:12]: This is a valid and correct value. Please see https://trac.macports.org/wiki/PortfileRecipes#license

**Review by mascguy — APPROVED** [review]

**Pull request merged**